.azure-pipelines/dotnet.yml

```diff
@@ -12,7 +12,7 @@ resources:
       name: internal/azure-sdk-build-tools
 
 variables:
-  DotNetCoreVersion: '2.1.503'
+  DotNetCoreVersion: '3.1.100'
```

Comment:
Is it the toolset or the framework you need here? I want to make sure that what we build doesn't preclude it from working when we consume it in our repo which is still using `3.0.100`.

Reply:
Let's check if `3.0.100` is enough.

Reply:
Yep, `3.0.100` is enough